omero/sysadmins/whatsnew.txt

```diff
@@ -72,3 +75,13 @@ OMERO.mail
 A new :doc:`additional feature <advanced-install>` has been added. :doc:`mail`
 allows you to configure the server to email users, for example to alert group
 owners to broken links in OMERO.web.
+
+OMERO permissions system
+------------------------
+
+Read-Write groups are now (as of OMERO version 5.1.2) supported in the
+clients. Existing groups can be promoted to this permissions level, which
+essentially allows all members to act as if they co-own the data except for
+the ability to move it between groups, via your usual OMERO admin tool. See
```

Comment:
Could we rearrange to not have the "what this permissions level does" in the middle of "you can promote groups ... with your usual tool"? Took me a moment to link the promote bit with the tool bit.